### DISCUSSION TITLE
Ability to import multiple GA properties (UA and/or GA4) into ONE domain or CSV-import

### DISCUSSION THREAD

**Discussion by zenminimalist** [plausible/analytics#3384] [Anything Else]
**Example**
Say I have Google UA data from 2020-2022 and from 2023-today have the GA4 data. Currently, it doesn't seem possible to import the delta from GA4 or import data from another GA property. 

However, having the ability to import multiple GA data properties (belonging to the same domain) as per example would be extremely helpful. 

Or at least have an option to import CSV in a pre-defined format. Give the user a template on how the data is imported. That way most use-cases could be satisfied with a "catch-all" if you will.

**Comment by Daniel15** [comment]
Plausible doesn't support GA4 at all at the moment, so that'd have to be added first.

**Comment by metmarkosaric** [comment]
The GA4 import is ready! You can see how it works here: https://plausible.io/docs/google-analytics-import

It’s now possible to import both UA and GA4 properties into the same Plausible dashboard. We’ll automatically detect time ranges to prevent double counting of periods and to avoid any gaps in the data.
